typescript/web/src/components/labeling-tool/openlayers-map/classification/classification-tag.tsx

```diff
@@ -28,7 +28,7 @@ export const ClassificationTag = ({
   labelClasses: LabelClassItem[];
   selectedLabelId: string | null;
   setSelectedLabelId: (labelId: string | null) => void;
-  client: ApolloClient<any>;
+  client: ApolloClient<object>;
```

Comment:
While it's cool to remove `any`, don't we really have something more strong?

Reply:
Ah sorry, I justified this change, but on the commit message, so it doesn't appear here.

First I went with `ApolloClient<NormalizedCacheObject>` which is the exact type of our clients.
However, it was conflicting with `useApolloClient` which returns an `ApolloClient<object>`.

So, I guessed it was safe to use this like this (this parameter only define the shaped of the cache).